### DISCUSSION TITLE
What ssl_options parameters are available?

### DISCUSSION THREAD

**Discussion by EmirDanfoss** [emqx/emqx#13966] [Q&A] [answered]
I am currently setting up my broker to ensure it's secure, but I can only find usage-specific information regarding the parameters for the ssl_options configuration block. Unfortunately, the documentation does not list all the available parameters.

Could you please provide a comprehensive list of all parameters that can be used within ssl_options, along with a brief description of each? This information would be very helpful for properly configuring SSL for my broker.

Thank you!

**Comment by thalesmg** [comment, marked as answer]
Hi, you can find the schema here in the configuration manual: https://docs.emqx.com/en/enterprise/v5.8.0/hocon/#V-listeners-S-listeners-ssl-S-mqtt_ssl_listener-ssl_options

**Reply by EmirDanfoss** [reply]
Missed this, thanks a bunch! :)
